eos-tech-support/eos-enable-coredumps

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 #
-# Enables coredumps and adjusts storage path to work with coredumpcl
+# Enables coredumps and adjusts storage path to work with coredumpctl.
+# This takes effect immediately without requiring a reboot.
```

Comment:
Good suggestion! Perhaps we should also note that this persists over reboot, and no crash metrics are collected once this is enabled?

Reply:
Also added the path where the coredumps are found.